config/initializers/doorkeeper.rb

```diff
@@ -47,8 +47,13 @@
   # Define access token scopes for your provider
   # For more information go to
   # https://github.com/doorkeeper-gem/doorkeeper/wiki/Using-Scopes
-  # default_scopes  :public
-  # optional_scopes :write, :update
+
+  # default is a collection of read-only scopes
+  default_scopes 'config:user', 'user:read'
+
+  optional_scopes 'config:read', 'config:write',
+                  'user:write',
+                  'offline_access'
```

Comment:
do we use `offline_access` already somewhere?

Reply:
It is used [here](https://github.com/foodcoops/foodsoft/blob/master/app/controllers/concerns/auth.rb#L138) (already on master).